test/Frame_spec.js

```diff
@@ -204,6 +204,18 @@ describe("Frame test",function(){
     expect(doc.documentElement.outerHTML).toEqual(renderedContent);
   });
 
+  it("should allow setting mountTarget", function () {
+    div = document.body.appendChild(document.createElement('div'));
+
+    var initialContent = '<!DOCTYPE html><html><head></head><body><h1>i was here first</h1><div id="mountHere"></div></body></html>';
+    var renderedContent = '<html><head></head><body><h1>i was here first</h1><div id="mountHere"><div data-reactid=".j"></div></div></body></html>';
+    var frame = ReactDOM.render(
+      <Frame initialContent={initialContent} mountTarget="#mountHere" />
```

Comment:
Add a child here since react 15 will drop the `data-reactid` attributes so you should check for something else.